srcs/app/http/HttpHeaders.cpp

```diff
@@ -0,0 +1,108 @@
+#include <HttpHeaders.hpp>
+#include <cctype>
```

Comment:
Do you agree that it is a good idea to place the additional includes for a source file in the header of its name?
I believed it makes code cleaner, but I've come across people with different opinions.

Reply:
I prefer to include only the necessary headers in the header files and the rest in the source files. But I see that we place all additional includes in the header files, and I will adhere to the same practice for consistency.